app/models/metric/chargeback_helper.rb

```diff
@@ -22,7 +22,15 @@ def tag_list_with_prefix
     if resource.kind_of?(Container)
       state = resource.vim_performance_state_for_ts(timestamp.to_s)
       image_tag_name = "#{state.image_tag_names}|" if state
-      labels = resource.try(:container_image).try(:docker_labels).try(:collect) { |l| "container_image/label/managed/#{l.name}/#{l.value}" }
+      labels = resource.try(:container_image).try(:docker_labels).try(:collect) do |l|
```

Comment:
Previously this built a list of strings, now a list of 2-item lists.
Perhaps you want s/collect/collect_concat/?